include/soft_limiter.h

```diff
@@ -230,7 +208,7 @@ void SL_TEMPLATE_INST::PrintStats(const AudioFrame<in_t> &external_scale) const
 	// More importantly, users typically care when their overall stream
 	// never achieved the maximum levels.
 	auto peak_ratio = peak_sample / upper_bound;
-	peak_ratio = std::min(peak_ratio, static_cast<in_t>(1.0));
+	peak_ratio = std::min(peak_ratio, static_cast<float>(1.0));
```

Comment:
`static_cast<float>(1.0)` -> `1.0f` (and similar change in line 220)

Reply:
Nice catch; got it.